packages/react-admin-stories/src/react-admin-core/FormSubmitErrors.tsx

```diff
@@ -1,8 +1,8 @@
 import { storiesOf } from "@storybook/react";
-import { FinalForm } from "@vivid-planet/react-admin-core";
-import { Field, FormPaper, Input } from "@vivid-planet/react-admin-form";
+import { FinalForm, form } from "@vivid-planet/react-admin";
```

Comment:
The subpage @vivid-planet/react-admin-form is kind of still there only the import syntax is different now.
now:
```ts
import { FinalForm, form } from "@vivid-planet/react-admin";
const { Field, FormPaper, Input } = form;
```
before:

```ts
import { FinalForm } from "@vivid-planet/react-admin-core";	
import { Field, FormPaper, Input } from "@vivid-planet/react-admin-form";	
```

Now for me is not clear what we are aiming for, 
do we want to remove 'sub-packages', then it seems natural to me to export all components at the same level like material-ui does:     

```ts
import { FinalForm, FormField, FormPaper, FormInput } from "@vivid-planet/react-admin";
```
or we use sub-directory imports

```ts
import { FinalForm } from "@vivid-planet/react-admin";
import {  Field, FormPaper, Input } from "@vivid-planet/react-admin/form";
```

Any other thougths?

Reply:
Or we do some grouping and indicate groups somehow (with $ for instance):

```ts
import { FinalForm, $form} from "@vivid-planet/react-admin";
const { Field, FormPaper, Input } = $form;

```

Reply:
Thanks for the input, see https://github.com/vivid-planet/react-admin/issues/241